src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/XamlMember.cs

```diff
@@ -147,10 +138,6 @@ internal XamlMember(EventInfo eventInfo, XamlSchemaContext schemaContext, XamlMe
         internal XamlMember(string attachablePropertyName, MethodInfo getter, MethodInfo setter,
             XamlSchemaContext schemaContext, XamlMemberInvoker invoker, MemberReflector reflector)
         {
-            if (attachablePropertyName == null)
```

Comment:
ValidateGetter and ValidateSetter are now called before checking attachablePropertyName for null and throwing the exception.  If ValidateGetter/Setter fail before the new check on line 153, the behavior is different (ArgumentNullException vs ArgumentException).